Examples/Examples-iOS/IGListKitExamples/SectionControllers/ListeningSectionController.swift

```diff
@@ -0,0 +1,58 @@
+/**
+ Copyright (c) 2016-present, Facebook, Inc. All rights reserved.
+
+ The examples provided by Facebook are for non-commercial testing and evaluation
+ purposes only. Facebook reserves all rights not expressly granted.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import IGListKit
+
+class ListeningSectionController: IGListSectionController, IGListSectionType, IncrementListener {
```

Comment:
Thoughts about closing down the internal variables of demo classes? Nothings stops idiots like me access and updating `var value: Int = 0` directly and assuming `things will just work`. Is that too nit picky? Why I ask is again that you guys are such a huge inspiration for us "lesser mortals" and pushing us in the right direction can only be a good thing :)